core/src/test/scala/org/allenai/common/EnumSpec.scala

```diff
@@ -40,8 +40,8 @@ class EnumSpec extends UnitSpec {
     }
   }
 
-  "Java serialization" should "work" in {
-    FakeEnum.all foreach { enum =>
+  it should "work with a constructor argument" in {
```

Comment:
I just realized I accidentally deleted a test here - there's supposed to be two `Java serialization` tests, but I accidentally deleted one.

I'm adding it back in & merging. :)